app/controllers/home_controller.rb

```diff
@@ -2,14 +2,23 @@ class HomeController < ApplicationController
     before_action :authenticate_user!
     # send back form to create   
     def new
+        @hi = "Hola2"
```

Comment:
What's the point of this instance variable? Can "hola" just be added to the view as text?